### PR TITLE
Adding example notebook and instructions for how to implement footer.

### LINKED ISSUE #92: Notebook style guide: Footer instruction confusing
https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md#footer says we need a footer with logo, but does not say how it should be implemented. As a result, we have spacetelescope/notebooks#71.

There needs to be a clear instruction on how the logo should be included, preferably in a way that does not require each notebook to provide its own logo file.

### PR DESCRIPTION
**Pull request by arfon**
Fixes #92

/ cc @kjbrooks @pllim

### CODE REVIEW

File: templates/example_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    "Describe the libraries we're using here. If there's something unusual, explain what the library is, and why we need it.\n",
+    "- *numpy* to handle array functions\n",
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *astropy.table Table* for creating tidy tables of the data\n",
+    "- *matplotlib.pyplot* for plotting data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```
Comment (pllim) [inline templates/example_notebook.ipynb]: Nitpick: Clear all outputs.

**Comment by arfon** [comment]
OK, I've updated the example notebook to try and make it more consistent. Could you take another look @pllim ?

File: templates/example_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## About this notebook\n",
+    "Let the world know who the author of this great notebook is! If possible/appropriate, include a contact email address for users who might need support (e.g. archive@stsci.edu)\n",
+    "\n",
+    "**Author:** Joe Blogs, Archive Scientist.  \n",
```
Comment (pllim) [inline templates/example_notebook.ipynb]: Gender-neutral name?

File: templates/example_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+    "**Updated On:** 08/10/2018"
```
Comment (pllim) [inline templates/example_notebook.ipynb]:
```suggestion
    "**Updated On:** YYYY-MM-DD"
```

File: templates/example_notebook.ipynb (hunk 2 above)
Comment (arfon) [reply templates/example_notebook.ipynb]: Fixed in d17c596

File: templates/example_notebook.ipynb (hunk 3 above)
Comment (arfon) [reply templates/example_notebook.ipynb]: Fixed in d17c596

**Comment by arfon** [comment]
> Maybe clear output again, or did you intentionally leave them in?

The one output I've left in shows the example output from the table. I think this is useful to leave this way?

File: templates/example_notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+    "## Exercises\n",
+    "Most notebooks are trying to convey _some_ information to their reader. It's often a good idea, if possible, to include in the notebook something where there is not an included answer so that the reader can cement in their mind how whatever it is they were supposed to learn from that notebook. If you do have exercise(s), be sure to leave a blank code cell underneath to show the user that it's meant for them to try it out right there. You may also want to include a \"solutions\" notebook next to your main notebook for the user to look at after they have finished their attempt."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Aditional Resources\n",
+    "While this isn't always necessary, sometimes you want to Provide some more resources for the reader who wants to learn something beyond what's in the notebook. Sometimes these don't exist, but if they do, it's good to put them at the end to give the reader somewhere else to go. Usually a list of links using markdown bullet-plus-link format is appropriate:\n",
```
Comment (pllim) [inline templates/example_notebook.ipynb]:
```suggestion
    "While this isn't always necessary, sometimes you want to provide some more resources for the reader who wants to learn something beyond what's in the notebook. Sometimes these don't exist, but if they do, it's good to put them at the end to give the reader somewhere else to go. Usually a list of links using markdown bullet-plus-link format is appropriate:\n",
```

**Comment by arfon** [comment]
Thanks for the review @pllim!

**Pull request merged**